File: sdk/csharp/libraries/microsoft.bot.builder.solutions.skills/SkillSwitchConfirmOption.cs
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

using Microsoft.Bot.Schema;

namespace Microsoft.Bot.Builder.Skills
{
    public class SkillSwitchConfirmOption
    {
        public Activity FallbackHandledEvent { get; set; }

        public string TargetIntent { get; set; }

        public Activity UserInputActivity { get; set; }
    }
}

File: sdk/typescript/libraries/botbuilder-skills/src/SkillSwitchConfirmOption.ts
/**
 * Copyright(c) Microsoft Corporation.All rights reserved.
 * Licensed under the MIT License.
 */

import { Activity } from 'botbuilder';

export interface ISkillSwitchConfirmOption {
    fallbackHandledEvent: Partial<Activity>;
    targetIntent: string;
    userInputActivity: Partial<Activity>;
}
